datahub-web-react/src/app/entity/shared/tabs/Documentation/components/DiscardDescriptionModal.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react';
+import { Modal, Button } from 'antd';
+import { useRouteToTab } from '../../../EntityContext';
+
+type Props = {
+    buttonProps?: Record<string, unknown>;
+    isDescriptionUpdated?: boolean;
+    urn: string;
+};
+
+export const DiscardDescriptionModal = ({ buttonProps, isDescriptionUpdated, urn }: Props) => {
+    const [isModalVisible, setIsModalVisible] = useState(false);
+    const routeToTab = useRouteToTab();
+    const localStorageDictionary = localStorage.getItem('editedDescriptions');
```

Comment:
This is the 4th time the raw string 'editedDescriptions' has appeared. If we change it in one place, we will have to change it in all 4. Please extract this into a shared constant.

Reply:
Checked and fixed.